### ISSUE TITLE
Workspace tab/button shows as a giant Icon in the top bar

### ISSUE THREAD

**Issue by msmafra** [Floorp-Projects/Floorp#739]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

Using the procedures highlighted in the issue #738, rendered in a giant icon on my top bar and it doesn't show the description "Workspace".

### To Reproduce

Follow what is in here https://blog.ablaze.one/3865/2024-01-17/

### Expected behavior

To not have a giant icon on my top bar and have the description "Workspace"

### Screenshots

![Captura de tela de 2024-01-19 12-32-00](https://github.com/Floorp-Projects/Floorp/assets/1005457/623c379d-b5b7-4a8e-ad7d-e4e5c9ba457e)
![Captura de tela de 2024-01-19 12-31-52](https://github.com/Floorp-Projects/Floorp/assets/1005457/e74aca0a-18ae-4905-863e-2b1fe57b113c)
![Captura de tela de 2024-01-19 12-45-45](https://github.com/Floorp-Projects/Floorp/assets/1005457/944442e1-3bdc-4f28-9960-fc52132cdcab)

### OS Info

Fedora Workstation 39

### Floorp Version

11.8.1

### Additional context

In the browser's about page it says 11.8.1 but Flatpak says 11.7.1:
```
❯ flatpak list | grep floo
Floorp  one.ablaze.floorp       11.7.1  stable  flathub user
```

**Comment by surapunoyousei** [comment]
deplicate and this issue will be fixed at 11.8.2 at next week

**Comment by surapunoyousei** [comment]
deplicate
